### PR TITLE
Release notes for 2022.10.0 monthly

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Add release notes for 2022.10.0. Update event type and oauth2 error codes for 2022.10.0
- **Is this PR related to a Monolith release?** Yes, 2022.10.0

### Resolves:

* [OKTA-537076](https://oktainc.atlassian.net/browse/OKTA-537076)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/release-notes/2022/index.md [hunk 1]
```diff
@@ -2,6 +2,74 @@
...
+### Monthly release 2022.10.0
+
+| Change | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [Additional measures for suspicious SMS and Voice requests](#additional-measures-for-suspicious-sms-and-voice-requests) | October 5, 2022 |
+| [Dynamic IdP routing is GA in Production](#dynamic-idp-routing-is-ga-in-production) | June 8, 2022 |
+| [Improved ThreatInsight coverage](#improved-threatinsight-coverage) | October 5, 2022 |
+| [Improved User API sort](#improved-user-api-sort) | October 5, 2022 |
+| [Non-deletable default authorization server is GA in Production](#non-deletable-default-authorization-server-is-ga-in-production) | August 31, 2022 |
+| [OAuth 2.0 authentication for inline hooks is Self-Service EA in Preview](#oauth-20-authentication-for-inline-hooks-is-self-service-ea-in-preview) | October 5, 2022 |
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022/index.md:16]: This link appears to be broken

File: packages/@okta/vuepress-site/docs/release-notes/2022/index.md [hunk 2]
```diff
@@ -2,6 +2,74 @@
...
+
+To improve the security of inline hooks, Okta now supports authentication with OAuth 2.0 access tokens. Tokens ensure secure calls to external web services.
+
+When creating inline hooks in the Admin Console (or by API), administrators or developers can now select OAuth 2.0 authentication and choose between two methods of OAuth 2.0: Client Secret or Private Key. A new [Key Management API](/docs/reference/api/hook-keys/) and Admin Console page is also available to create public/private key pairs for use with OAuth 2.0 inline hooks. See [Key management](https://help.okta.com/okta_help.htm?type=oie&id=ext-key-management).
+
+Using the OAuth 2.0 framework provides better security than Basic Authentication or custom headers, and is less work than setting up an IP allowlisting solution. Clients also have the ability to use access tokens minted by their own custom authorization servers to guarantee that Okta is calling their client web services and isn't triggered by any external actors. See [Add an event hook](https://help.okta.com/okta_help.htm?type=oie&id=ext-add-event-hooks). <!--OKTA-537306-->
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022/index.md:49]:
```suggestion
Using the OAuth 2.0 framework provides better security than Basic Authentication or custom headers, and is less work than setting up an IP allowlisting solution. Clients also have the ability to use access tokens minted by their own custom authorization servers to guarantee that Okta is calling their client web services and isn't triggered by any external actors. See [Add an inline hook](https://help.okta.com/okta_help.htm?type=oie&id=ext-add-inline-hook). <!--OKTA-537306-->
```

Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2022/index.md:49]: Incorrect h.o.c link.

File: packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md [hunk 3]
```diff
@@ -4,6 +4,110 @@ title: Okta Identity Engine API Products release notes 2022
...
+| [Default authenticator enrollment policy settings format is GA in Production](#default-authenticator-enrollment-policy-settings-format-is-ga-in-production) | August 31, 2022 |
+| [Dynamic IdP routing is GA in Production](#dynamic-idp-routing-is-ga-in-production) | June 8, 2022 |
+| [Factors API support for Okta Verify authenticator enrollment flows is GA in Preview](#factors-api-support-for-okta-verify-authenticator-enrollment-flows-is-ga-in-preview) | October 5, 2022 |
+| [Improved ThreatInsight coverage](#improved-threatinsight-coverage) | October 5, 2022 |
+| [Improved User API sort](#improved-user-api-sort) | October 5, 2022 |
+| [Manage embedded widget sign-in support is EA in Preview](#manage-embedded-widget-sign-in-support-is-ea-in-preview) | October 5, 2022 |
+| [Non-deletable default authorization server is GA in Production](#non-deletable-default-authorization-server-is-ga-in-production) | August 31, 2022 |
+| [OAuth 2.0 authentication for inline hooks is Self-Service EA in Preview](#oauth-20-authentication-for-inline-hooks-is-self-service-ea-in-preview) | October 5, 2022 |
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md:23]: link didn't work for this one again

File: packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md [hunk 4]
```diff
@@ -4,6 +4,110 @@ title: Okta Identity Engine API Products release notes 2022
...
+
+To improve the security of inline hooks, Okta now supports authentication with OAuth 2.0 access tokens. Tokens ensure secure calls to external web services.
+
+When creating inline hooks in the Admin Console (or by API), administrators or developers can now select OAuth 2.0 authentication and choose between two methods of OAuth 2.0: Client Secret or Private Key. A new [Key Management API](/docs/reference/api/hook-keys/) and Admin Console page is also available to create public/private key pairs for use with OAuth 2.0 inline hooks. See [Key management](https://help.okta.com/okta_help.htm?type=oie&id=ext-key-management).
+
+Using the OAuth 2.0 framework provides better security than Basic Authentication or custom headers, and is less work than setting up an IP allowlisting solution. Clients also have the ability to use access tokens minted by their own custom authorization servers to guarantee that Okta is calling their client web services and isn't triggered by any external actors. See [Add an event hook](https://help.okta.com/okta_help.htm?type=oie&id=ext-add-event-hooks). <!--OKTA-537306-->
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md:83]:
```suggestion
Using the OAuth 2.0 framework provides better security than Basic Authentication or custom headers, and is less work than setting up an IP allowlisting solution. Clients also have the ability to use access tokens minted by their own custom authorization servers to guarantee that Okta is calling their client web services and isn't triggered by any external actors. See [Add an inline hook](https://help.okta.com/okta_help.htm?type=oie&id=ext-add-inline-hook). <!--OKTA-537306-->
```

File: packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md [hunk 5]
```diff
@@ -4,6 +4,110 @@ title: Okta Identity Engine API Products release notes 2022
...
+| [Improved User API sort](#improved-user-api-sort) | October 5, 2022 |
+| [Manage embedded widget sign-in support is EA in Preview](#manage-embedded-widget-sign-in-support-is-ea-in-preview) | October 5, 2022 |
+| [Non-deletable default authorization server is GA in Production](#non-deletable-default-authorization-server-is-ga-in-production) | August 31, 2022 |
+| [OAuth 2.0 authentication for inline hooks is Self-Service EA in Preview](#oauth-20-authentication-for-inline-hooks-is-self-service-ea-in-preview) | October 5, 2022 |
+| [Developer documentation updates in 2022.10.0](#developer-documentation-updates-in-2022-10-0) | October 5, 2022 |
+| [Bugs fixed in 2022.010.0](#bugs-fixed-in-2022-10-0) | October 5, 2022|
+
+#### Access Denied error message customization is GA in Production
+
+Admins can now customize the error message that users receive when their access is denied. This allows admins to provide remediation steps and/or point users to documentation that will help resolve their access issues. <!--OKTA-512725-->
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md:29]: "... to documentation that helps resolve their access issues"

**Review by brianduffield-okta — APPROVED** [review]
A few small items. Approving not to hold you up. Great job, Van.

File: packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md (hunk 5 above)
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md]: Updated. Thanks!

File: packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md (hunk 3 above)
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md]: fixed

File: packages/@okta/vuepress-site/docs/release-notes/2022/index.md (hunk 1 above)
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2022/index.md]: fixed

**Pull request merged**